service2/server.js

```diff
@@ -0,0 +1,24 @@
+const express = require("express");
+const axios = require("axios");
+const app = express();
+
+const PORT = process.env.PORT || 3001;
+
+// This says to listen for a GET request on the route /aRoute'
+app.get("/aRoute", (req, res) => {
+  // This will perform a GET request to service1
+  axios
+    .get(process.env.SERVICE_ADDRESS)
```

Comment:
Would be good to include a default (`http://localhost:3000`).
Then people don't need to set up environment variables for running locally (unless you want to explain to everyone how to set it up to run locally).
See how I've done it with the port number at the top